docker-compose.yml

```diff
@@ -21,6 +21,7 @@ slave:
         - jenkins
     volumes:
         - ./slave:/home/slave
+        - /ome:/ome
```

Comment:
This will likely prevent deployment on orca-3, but that I can fix locally if need be.